.eslintrc

```diff
@@ -25,6 +30,12 @@
     "import/internal-regex": "^@18f/identity-"
   },
   "overrides": [
+    {
+      "files": ["*"],
+      "rules": {
+        "react/react-in-jsx-scope": "off"
+      }
+    },
```

Comment:
Rather than defining this as an override, I think we can put the rule in the top-level `"rules"` (line 16 above) and it'd work just the same?

Reply:
✔️ Fixed that. I'll push a commit shortly.

Reply:
[Done](https://github.com/18F/identity-idp/pull/9706/commits/850811704446194c836de40259cb5517a794ffad).